lib/workers/repository/updates/generate.ts

```diff
@@ -148,8 +148,12 @@ export function generateBranchConfig(
     // Delete group config regardless of whether it was applied
     delete upgrade.group;
 
-    // istanbul ignore else
```

Comment:
why do you remove this? are you sure that branch is covered? otherwise it will decrease branch coverage.

Reply:
I think it's covered. Otherwise wouldn't we get a CI failure now? I thought better to remove ignored where possible

Reply:
no, you don't get a CI failures, because the else block isn't there. so it's just a not covered branch, which we try to get to 100% too

Reply:
Is it a problem? I thought we want to minimize Istanbul ignore statements

Reply:
no, it will reduce coverage. so in this case we should leave it or make a test for the else case

Reply:
but shouldn't we be avoiding using istanbul ignore statements to increase coverage of reachable code anyway?

Reply:
sure, but then we should have a test case for the else case, even if it's empty.